### PR TITLE
Altitude in captcha solving and quick fixes.

### PR DESCRIPTION
**Pull request by neskk**
## Description

Added altitude support to requests sent from captcha solvers.
Some quick text fixes and a bit of cleanup.
Documentation for captchas
Manual/Hybrid captcha mode major bug fixes:
- using `status['username']` was a brainless mistake. sorry...
- using `status['last_scan_time']` was also a bad idea...

Some status about accounts getting uncaptcha'd will never be visible in status page because the worker is using another account and updating its status.
Inject.js - captcha solving page now updates stats when it refreshes.

## How Has This Been Tested?

Tested in both Windows and Linux machines and tests went well.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/extras/commandline.md [hunk 1]
```diff
@@ -7,7 +7,9 @@
                         [-wph WORKERS_PER_HIVE] [-l LOCATION] [-nj]
                         [-st STEP_LIMIT] [-sd SCAN_DELAY]
                         [--spawn-delay SPAWN_DELAY] [-enc] [-cs] [-ck CAPTCHA_KEY]
-                        [-cds CAPTCHA_DSK] [-ed ENCOUNTER_DELAY]
+                        [-cds CAPTCHA_DSK] [-mcd MANUAL-CAPTCHA-DOMAIN]
+                        [-mcr MANUAL-CAPTCHA-REFRESH]
+                        [-mct MANUAL-CAPTCHA-TIMEMOUT] [-ed ENCOUNTER_DELAY]
```
Comment (sLoPPydrive) [inline docs/extras/commandline.md]: It's "timeout", not "timemout" ;)

File: docs/extras/commandline.md [hunk 2]
```diff
@@ -105,6 +107,16 @@
       -cds CAPTCHA_DSK, --captcha-dsk CAPTCHA_DSK
                             PokemonGo captcha data-sitekey. [env var:
                             POGOMAP_CAPTCHA_DSK]
+      -mcd MANUAL-CAPTCHA-DOMAIN --manual-captcha-domain
+                            Domain to where captcha tokens will be sent
+                            (default: http://127.0.0.1:5000)
+      -mcr MANUAL-CAPTCHA-REFRESH --manual-captcha-refresh
+                            Time available before captcha page refreshes
+                            (default: 30 seconds)
+      -mct MANUAL-CAPTCHA-TIMEMOUT -manual-captcha-timeout
```
Comment (sLoPPydrive) [inline docs/extras/commandline.md]: Again the time(m)out...

Comment (neskk) [inline docs/extras/commandline.md]: Thanks.

**Comment by Glennmen** [comment]
@neskk Is there a reason why it is refreshing the entire page and not only the stats?

**Comment by neskk** [comment]
@Glennmen: yes, to refresh the captcha.

**Comment by Glennmen** [comment]
I understand that it refreshes the captcha, but is it necessary? Does the captcha expire after a given time or can you disable the page refresh and only refresh the stats?
If it is necessary to refresh the page what time would you recommend for refresh time?

**Comment by neskk** [comment]
@Glennmen In our tests we deemed it necessary...
https://rocketmap.readthedocs.io/en/develop/extras/captchas.html
If you want try it out, experiment with different refresh times but please don't ask questions you can answer on your own.

**Pull request merged**